workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx

```diff
@@ -114,7 +110,8 @@ export function NextStepPage({
 		[TypeLicense.PAID]: (
 			<Header
 				description={
-					<>
+				
+						<>
 						<p>
```

Comment:
could you format this piece of code, please?
```suggestion
				
					<>
						<p>
```